backend/graminsta/post/serializers.py

```diff
@@ -70,3 +57,22 @@ class Meta:
         """
         model = get_user_model()
         fields = ['id', 'first_name', 'last_name', 'is_following']
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    """
+    Serializer that serializes Comment object
+    """
+    publisher = UserSerializer(required=True)
```

Comment:
Maybe you can try something like 
```first_name = serializers.CharField(source='publisher.first_name')```
so that you do not need to include a user object in comment object at frontend, since first_name and last_name would be enough to show the comment? But it's okay now!

Reply:
+1